riscv-zig-blink/build.zig

```diff
@@ -0,0 +1,43 @@
+const std = @import("std");
+const Builder = std.build.Builder;
+
+pub fn build(b: *Builder) void {
+    b.setPreferredReleaseMode(.ReleaseSmall);
+    const mode = b.standardReleaseOptions();
+
+    const elf = b.addExecutable("riscv-zig-blink", "src/main.zig");
+    elf.setTheTarget(.{
+        .Cross = .{
+            .arch = .riscv32,
+            .os = .freestanding,
+            .abi = .none,
+            .cpu_features = .{
+                .cpu = &std.Target.riscv.cpu.generic_rv32,
+                .features = std.Target.Cpu.Feature.Set.empty,
+            },
+        },
+    });
+    elf.setLinkerScriptPath("ld/linker.ld");
+    elf.setBuildMode(mode);
+    elf.setOutputDir(".");
```

Comment:
Wasn't sure if I should emit the .elf in the current directory or not. We need it for the `gdb` flow, but not otherwise...